app/story/route.js

```diff
@@ -38,25 +38,42 @@ export default Ember.Route.extend(PlayParamMixin, {
       transition.send('updateDonateChunk', get(model, 'story.headerDonateChunk'));
     }
   },
-  
+
   setupController(controller) {
     controller.set('isMobile', window.Modernizr.touchevents);
     controller.set('session', get(this, 'session'));
     controller.set('user', get(this, 'currentUser.user'));
     return this._super(...arguments);
   },
-  
+
+  renderTemplate(controller, model) {
+    if (get(model, 'story.template') === 'story_full-bleed') {
+      this.send('disableChrome');
+      let image = get(model, 'story.imageMain');
+      if (image){
```

Comment:
this logic makes sense but could use a little comment explaining the three 3 cases for `showShareLinks`